app/models/groupe_instructeur.rb

```diff
@@ -75,6 +75,10 @@ def can_delete?
     dossiers.empty? && (procedure.groupe_instructeurs.active.many? || (procedure.groupe_instructeurs.active.one? && closed))
   end
 
+  def can_close?
+    self != procedure.defaut_groupe_instructeur
```

Comment:
```suggestion
    id != procedure.defaut_groupe_instructeur_id
```

Ça évite de charger `defaut_groupe_instructeur` s'il n'est pas déjà chargé